CMakeLists.txt

```diff
@@ -581,6 +581,12 @@ install(
     DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/applications
 )
 
+install(
+    FILES
+    ${CMAKE_SOURCE_DIR}/freeorion.appdata.xml
+    DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/appdata
```

Comment:
In fact I am not sure. I thought metainfo was for plugins/addons. Something must have changed there. Maybe @hughsie can explain current best practice.